### DISCUSSION TITLE
Growing an Open Source Project

### DISCUSSION THREAD

**Discussion by Rishit-dagli** [github/india#40] [Open Source]
I am watching the awesome [Maintaining large open-source projects](https://githubsatellite.com/sessions/maintaining-large-open-source-projects/) session and I am already loving it ❤️ . I had a question about growing open-source projects, how one should be going about doing so, any tips? I would love to incorporate any advice for my own projects!

**Comment by karthiksedoc** [comment]
you can read about it in [GitHub page for Open Source](https://github.com/open-source) 👍

**Comment by FRTMentor** [comment]
Please visit the GitHub site, try to create an account similar to the FRT.
And as GitHub landing page will guide to create and and ad the repository.
Also please go through some GitHub learning video's or google links before creating.

**Comment by izam-mohammed** [comment]
### Growing an open-source project can be a challenging task, but there are several strategies that can be effective in attracting and retaining contributors. Some tips include:

- Clearly communicate the project's goals and vision. This will help attract potential contributors who share the same values and are passionate about the project.
- Make it easy for people to get started with the project. This includes providing clear documentation, tutorials, and examples.
- Encourage and facilitate collaboration. This includes providing clear guidelines and processes for contributing, and actively seeking out contributions from the community.
- Create a welcoming and inclusive community. This includes fostering a positive and supportive culture, and being responsive and respectful when interacting with contributors.
- Leverage social media and other online platforms to promote the project and engage with the community.
- Make sure your project is well-maintained and keeps up to date with the latest technologies and best practices.
- Be transparent with your decision-making process, and have open communication channels available for the community.
- Reward and recognize contributions, whether it's through providing swags, badges, or opportunities for leadership roles, this will help to keep the community motivated.

Remember that growing a successful open-source project takes time and effort, but by following these tips, you can create a thriving community of contributors and users.
